Sprint-2/implement/tally.js

```diff
@@ -1,3 +1,15 @@
-function tally() {}
+function tally(items) {
+    const frequencies = {};
+
+    if (typeof items === "string") {
+        throw new Error("Invalid input");
+    }
```

Comment:
If `items` is not a string, it could still be other types of value (in addition to array). How should we check if `items` is an array in JavaScript?